bedrock/mozorg/templates/mozorg/advertising/contact.html

```diff
@@ -0,0 +1,60 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
```

Comment:
Is this page redundant? When I go  to https://www-demo1.allizom.org/en-US/advertising/contact/ I see a form that goes a GET and there is no backend view to handle it and we're not posting to basket via form-action or JS.

If it's not a working page, we shouldn't expose it.